portal/templates/redesign/home_new.html

```diff
@@ -19,18 +19,18 @@
 
 <div class="row row--regular row--flex">
     <section class="banner--homepage banner--homepage--teacher col-sm-6">
-        <div class="banner col-sm-6 col-sm-offset-4">
+        <div class="banner col-sm-7 col-sm-offset-3">
```

Comment:
This is a good example of why I don't approve of using classnames like these directly in your markup